chain-signatures/node/src/storage/error.rs

```diff
@@ -1,13 +0,0 @@
-pub type StoreResult<T> = std::result::Result<T, StoreError>;
-
-#[derive(Debug, thiserror::Error)]
-pub enum StoreError {
-    #[error("redis error: {0}")]
-    Redis(#[from] redis::RedisError),
-    #[error("storage connection error: {0}")]
-    Connect(#[from] anyhow::Error),
-    #[error("empty: {0}")]
-    Empty(&'static str),
-    #[error("other: {0}")]
-    Other(String),
-}
```

Comment:
Hm, I guess we can remove these to simplify. One could argue that it makes things a bit less testable, if we wanted to add more detailed tests on how we behave with unexpected storage states etc. But I like your approach to remove what we are not currently using. :+1: Applying the YAGNI principle after the fact :D

side note: combining multiple significant changes or refactorings in one PR makes it quite a bit harder for reviewers, which in turn makes it easier for errors to slip through. I usually try to make separate PRs or at least keep it in clearly separated commits and let reviewers know that they should review commit-by-commit.